tests/core/test_scripts.py

```diff
@@ -2564,6 +2577,194 @@ def test_suppress_work(self, db: DatabaseTransactionFixture):
         assert work.suppressed_for == [test_library]
 
 
+class TestGenerateInventoryReports:
+    def test_do_run(self, db: DatabaseTransactionFixture):
+        # create some test data that we expect to be picked up in the inventory report
+        library = db.library(short_name="test")
+        settings = OPDSImporterSettings(
+            include_in_inventory_report=True,
+            external_account_id="http://opds.com",
+            data_source="BiblioBoard",
+        )
+        collection = db.collection(
+            name="BiblioBoard Test Collection", settings=settings.dict()
+        )
+        collection.libraries = [library]
+
+        # Configure test data we expect will not be picked up.
+        no_inventory_report_settings = OPDSImporterSettings(
```

Comment:
I added this second collection/datasource  to ensure we are not picking up collections that should be out of scope.